Mena_4pm_ProjectPortfolio_Team12.c

```diff
@@ -96,10 +102,66 @@ void DisplayBoard(int board[][7]) {
         printf("\n\n\n\n\n");
     }
 }
-int CheckNorth();
-int CheckEast();
-int CheckSouth();
-int CheckWest();
+int CheckNorth(int row, int col, int board[][7]) {
+    int count = 0;
+    int i = row;
+    if (i == 0) {
+        return 1;
+    }
+    while (board[i][col] == board[i-1][col]) {
+        count++;
+        i--;
+        if (i == 0) {
+            break;
+        }
+    }
+    return count;
+}
```

Comment:
I changed the statements in the if statements to be just == 0 since that means you're at the top of the array and cant go more north than that. i also changed the intital retun value to be 1 instead of 0